vms/volunteer/templates/volunteer/edit.html

```diff
@@ -229,9 +231,9 @@
                     </div>
                 </div>
                 {% if form.unlisted_organization.errors %}
-                    <div class="form-group has-error">
+                    <div id="div_id_unlisted_organization" class="form-group has-error">
                         <label class="col-md-2 control-label">{% trans "If your organization is not listed, please provide it here:" %}</label>
-                        <div class="col-md-8">
+                        <div id="div_id_unlisted_organization" class="col-md-8">

```

Comment:
oopsie. same `id` used twice ?

Reply:
@tonythomas01 If I change the id then I will have to rewrite the hide_organization function again for that which wil be redundant.  
![image](https://user-images.githubusercontent.com/22369767/36341296-1a0de3ee-1412-11e8-94e7-ceef26618885.png)
Can't we keep the same id as they are in different templates and both have exactly same function?

Reply:
even though, I am wondering why you have to use this id on these two fields. When you hide the parent element, the child too gets hidden right ? 

Bonus, this hide and seek in js can be replaced with http://api.jquery.com/toggle/

Reply:
@tonythomas01 That is not the parent infact they reflect the same div  that's why they have the same id. There is an if-else statement in between.
![screenshot from 2018-02-23 21-31-08](https://user-images.githubusercontent.com/22369767/36603481-e2aba246-18e0-11e8-815b-2226ec46f9d0.png)